kythe/go/extractors/config/runextractor/compdb/compdb_test.go

```diff
@@ -38,10 +38,7 @@ const (
 func setupEnvironment(t *testing.T) string {
 	t.Helper()
 	// TODO(shahms): ExtractCompilations should take an output path.
-	output := os.Getenv("TEST_TMPDIR")
-	if output == "" {
-		t.Skip("Skipping test due to incompatible environment (missing TEST_TMPDIR)")
-	}
+	output := t.TempDir()
```

Comment:
Changing this because otherwise the second run of testExtractCompilationsEndToEndWithDatabase will try to overwrite a generated file and fail.

Reply:
This works (and is documented to do so by Bazel as well).